scripts/python-skywater-pdk/skywater_pdk/liberty.py

```diff
@@ -166,6 +180,34 @@ def top_corner_file(libname, corner, corner_type: TimingType):
         corner=corner, corner_type=corner_type.file)
 
 
+def top_liberty_file(libname, corner, corner_type: TimingType):
```

Comment:
How is this different to the `top_corner_file` function above?

Reply:
`.lib` vs `.lib.json`

Reply:
Maybe `def top_liberty_file` should be `def top_liberty_file_json` named?

Reply:
Huh ?  No the `top_liberty_file` is the one _without_ `json` in its name
`top_corner_file` is the JSON file.